lib/ansible/galaxy/collection.py

```diff
@@ -461,6 +461,19 @@ def validate_collection_name(name):
                        "characters from [a-zA-Z0-9_] only." % name)
 
 
+def validate_collection_path(collection_path):
+    """ Ensure a given path ends with 'ansible_collection'
```

Comment:
Docstring and code don't match.  Guessing there should be an `s` here and the next 2 mentions.

Reply:
Fixed.